### PR TITLE
Parameterize machine names in edx-west playbooks

### PR DESCRIPTION
**Pull request by sefk**
This is pulling into master a few changes we've made in the playbooks we use to drive our own Stanford instance.  Mostly want to keep master in sync so others can see what we're doing and our upcoming merges are cleaner.

@feanil -- can you take a look?  Just asking you since you've been looking at other of my changes, but this one isn't very exciting. 

Look at PR #854 for discussion with Joe and Jason about the over all feature.  Thanks.

### CODE REVIEW

File: playbooks/edx-west/carnegie-prod-app.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
+<<<<<<< HEAD
```
Comment (feanil) [inline playbooks/edx-west/carnegie-prod-app.yml]: Looks like a merge conflict that wasn't resolved.

**Comment by sefk** [comment]
whoops @feanil sorry i missed that.  Fixed now.

**Comment by feanil** [comment]
looks good.

**Comment by sefk** [comment]
Appreciate the extra set of eyes @feanil -- pulling.

**Pull request merged**